addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -69,7 +69,7 @@ function getColorName(value) {
  *  exist
  * @returns {string}
  */
-function getImageSrcFromRecordInfo(record, model, field, idOrIds, placeholder) {
+export function getImageSrcFromRecordInfo(record, model, field, idOrIds, placeholder) {
```

Comment:
Maybe we should not export them from kanban_record, but from view utils, and import them in both views, which reduces the idea that kanban is the main view

Reply:
i would keep it here for now